app-router/installation.mdx

```diff
@@ -0,0 +1,168 @@
+import ProvidersExample from "/snippets/app-router/providers.mdx"
+import LayoutExample from "/snippets/app-router/layout.mdx"
+import ApiHandlerExample from "/snippets/app-router/api-handler.mdx"
+import RuntimeExample from "/snippets/reference/runtime.mdx"
+import ClientExample from "/snippets/reference/client.mdx"
+import NextConfigExample from "/snippets/reference/next-config.mdx"
+
+The fastest way to get started with Makeswift on a new Next.js project is to follow the
+[quickstart](/quickstart) guide. If you have an existing Next.js application
+or want to set things up yourself, continue with the rest of this guide.
+
+## System requirements
+
+- [Node.js 18.17](https://nodejs.org/en) or a later version.
+- macOS, Windows (including WSL), and Linux are supported.
+
+## Geting started
+
+<Steps>
+  <Step title="Open your Next.js project">
+    First, open your Next.js project. If you don't already have one, head over to
+    the [Next.js](https://nextjs.org/docs/getting-started/installation)
+    documentation to get one set up. If you do have one, please verify you are
+    using [Next.js 13.4](https://nextjs.org/blog/next-12) or a later version that is using app router.
+
+    <Info>
+      If you are not using app router, here's how to [incrementally adopt](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration) it.
+    </Info>
+
+  </Step>
+  <Step title="Install dependencies">
+  Install the `@makeswift/runtime` package. This package contains all of the necessary
+  code to integrate Makeswift into your Next.js app.
+
+```bash
+npm install @makeswift/runtime
+```
+
+  </Step>
+  <Step title="Add API key to environment variables">
+  
+Requesting data through the `Makeswift` client requires a site API key from Makeswift. In the Makeswift builder, go to **Settings > Host** and copy the API key for the site.
+
+<Frame>
+  <img src="/images/site-api-key.gif" alt="How to get the site API key" />
+</Frame>
+
+Once the API key is in your clipboard, open your [`.env.local`](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables) file and paste the snippet below.
+
+```sh
+MAKESWIFT_SITE_API_KEY=paste-your-api-key-here
+```
+
+  </Step>
+  <Step title="Add Makeswift runtime and client">
+  
+Create files for the Makeswift [runtime](/reference/runtime/constructor) and [client](/reference/client/constructor).
+
+<CodeGroup>
+  <RuntimeExample />
+  <ClientExample />
+</CodeGroup>
+
+</Step>
+  <Step title="Add the Makeswift API handler">
+  
+Similar to [NextAuth.js](https://next-auth.js.org/), Makeswift uses an API handler to communicate with your Next.js app. Create the file `app/api/makeswift/[...makeswift]/route.ts`. It's important that this directory and file structure is followed precisely.
+
+<ApiHandlerExample />
+
+This API route adds support for
+[preview Mode](https://nextjs.org/docs/pages/building-your-application/configuring/preview-mode),
```

Comment:
(Blocking) App router apps will actually be using draft mode now. Shoutout to @arvinpoddar!

I would just change this label and link to point to those docs

Reply:
handled